### ISSUE TITLE
Fine Tuning of GPT-3 codex and CMock

### ISSUE THREAD

**Issue by JohanHardy** [ThrowTheSwitch/Unity#651]
Everything is in the title. is there a change to start this topic @throwTheSwitch ?

**Comment by AndersonMartins1** [comment]
No, there is no specific change required to start a thread with a mention. You can simply start your thread or post by mentioning @ThrowTheSwitch followed by the content you want to discuss. This will notify him of your post and he can join the conversation if he wishes

**Comment by mvandervoord** [comment]
Closing whatever this was. Feel free to open a topic containing actual questions or feature requests.
